nuget/OpenTelemetry.AutoInstrumentation.nuspec

```diff
@@ -0,0 +1,68 @@
+        <dependency id="OpenTelemetry.Api" version="1.3.1" />
+        <dependency id="OpenTelemetry.Exporter.Console" version="1.3.1" />
+        <dependency id="OpenTelemetry.Exporter.Jaeger" version="1.3.1" />
+        <dependency id="OpenTelemetry.Exporter.OpenTelemetryProtocol" version="1.3.1" />
+        <dependency id="OpenTelemetry.Exporter.Prometheus" version="1.3.0-rc.2" />
+        <dependency id="OpenTelemetry.Exporter.Zipkin" version="1.3.1" />
+        <dependency id="OpenTelemetry.Extensions.Propagators" version="1.3.1" />
+        <dependency id="OpenTelemetry.Instrumentation.GrpcNetClient" version="1.0.0-rc9.4" />
+        <dependency id="OpenTelemetry.Instrumentation.Http" version="1.0.0-rc9.4" />
+        <dependency id="OpenTelemetry.Instrumentation.Runtime" version="1.0.0" />
+        <dependency id="OpenTelemetry.Instrumentation.SqlClient" version="1.0.0-rc9.4" />
+        <dependency id="OpenTelemetry.Instrumentation.Wcf" version="1.0.0-rc.7" />
+        <dependency id="OpenTelemetry.Shims.OpenTracing" version="1.0.0-rc9.4" />
+        <dependency id="OpenTelemetry.Instrumentation.AspNetCore" version="1.0.0-rc9.4" />
+        <dependency id="OpenTelemetry.Exporter.OpenTelemetryProtocol.Logs" version="1.3.0-rc.2" />
+      </group>
+      <group targetFramework=".NETFramework4.6.2">
+        <dependency id="OpenTelemetry" version="1.3.1" />
+        <dependency id="OpenTelemetry.Api" version="1.3.1" />
+        <dependency id="OpenTelemetry.Exporter.Console" version="1.3.1" />
+        <dependency id="OpenTelemetry.Exporter.Jaeger" version="1.3.1" />
+        <dependency id="OpenTelemetry.Exporter.OpenTelemetryProtocol" version="1.3.1" />
+        <dependency id="OpenTelemetry.Exporter.Prometheus" version="1.3.0-rc.2" />
+        <dependency id="OpenTelemetry.Exporter.Zipkin" version="1.3.1" />
+        <dependency id="OpenTelemetry.Extensions.Propagators" version="1.3.1" />
+        <dependency id="OpenTelemetry.Instrumentation.GrpcNetClient" version="1.0.0-rc9.4" />
+        <dependency id="OpenTelemetry.Instrumentation.Http" version="1.0.0-rc9.4" />
+        <dependency id="OpenTelemetry.Instrumentation.Runtime" version="1.0.0" />
+        <dependency id="OpenTelemetry.Instrumentation.SqlClient" version="1.0.0-rc9.4" />
+        <dependency id="OpenTelemetry.Instrumentation.Wcf" version="1.0.0-rc.7" />
+        <dependency id="OpenTelemetry.Shims.OpenTracing" version="1.0.0-rc9.4" />
+        <dependency id="OpenTelemetry.Instrumentation.AspNet" version="1.0.0-rc9.5" />
+      </group>
+    </dependencies>
+    <frameworkAssemblies>
+      <frameworkAssembly assemblyName="System.Configuration" targetFramework=".NETFramework4.6.2" />
+      <frameworkAssembly assemblyName="System.Web" targetFramework=".NETFramework4.6.2" />
+      <frameworkAssembly assemblyName="System.ServiceModel" targetFramework=".NETFramework4.6.2" />
+    </frameworkAssemblies>
+  </metadata>
+  <files>
```

Comment:
What's about alpine stuff?

Reply:
Both Linux variation share the same name for native code, we could accommodate alpine only if we create a separate folder in the NuGet package to hold `OpenTelemetry.AutoInstrumentation.Native.so`.

Reply:
We could scope this PR to solve Windows, Ubuntu and Mac-OS and follow up alpine in a different issue as we are not offering this Nuget as release bits for beta 4.

I feel the best option to accommodate Alpine is through renaming our  native libraries like `OpenTelemetry.AutoInstrumentation.Linux.Native.so`, `OpenTelemetry.AutoInstrumentation.Alpine.Native.so`, etc., Better to consider this in a separate issue.

Reply:
> I feel the best option to accommodate Alpine is through renaming our native libraries like OpenTelemetry.AutoInstrumentation.Linux.Native.so, OpenTelemetry.AutoInstrumentation.Alpine.Native.so, etc., Better to consider this in a separate issue.

It could be `OpenTelemetry.AutoInstrumentation.Native.musl.so` and the "regular Linux" could be `OpenTelemetry.AutoInstrumentation.Alpine.Native.glibc.so.

Reply:
Or separate folders `/linux-musl` and `/linux-glibc` like we have `win-x86` and `win-x64` for Windows.

Reply:
Having separate folders instead of file names seems better in case we have more native files in the future, anyway, to be decided later. Follow-up on #1397